### DISCUSSION TITLE
Any issues with the hue button of the new Philips hue dimmer switch V2 ?

### DISCUSSION THREAD

**Discussion by ceysa75** [Koenkk/zigbee2mqtt#7124] [Devices]
Hi all,

at several places I have read that there are issues with zigbee2mqtt and the hue button (see fig. below) of the new dimmer switch of Philiips (929002398602). According to the devices section of zigbee2mqtt at least it is not reported that any problems exist. Is anyone here is using that switch without any issues ?

Thx and best regards 
![image](https://user-images.githubusercontent.com/37417641/115158940-02a4f800-a091-11eb-8bcd-a1bb553e7ec2.png)

**Comment by GD-Dal** [comment]
The button itself works fine.
But they have added a fooking annoying function to it:
When I press one of the buttons with the intention to hold it down for a while, it first sends a "down_press" before starting the "down_hold repetition"
So if you have a flow where you have set up that a short press turns off the light immediately, and a long press dims the light, it will not work anymore. It just turns off the light straight away.

This is the same for all 4 buttons.

**Reply by ceysa75** [reply]
Thank you for the explanation. I just have ordered one and will see if I can live with that. Out of curiosity: Any chance that this will be changed in future firmware updates by Philips?
